src/json.ts

```diff
@@ -266,3 +272,93 @@ export function getJsonRpcIdValidator(options?: JsonRpcValidatorOptions) {
   };
   return isValidJsonRpcId;
 }
+
+/**
+ * Checks whether a value is JSON serializable and counts the total number
+ * of bytes needed to store the serialized version of the value.
+ *
+ * This function assumes the encoding of the JSON is done in UTF-8.
+ *
+ * @param value - A potential JSON serializable value.
+ * @param skipSizing - Skip JSON size calculation (default: false).
+ * @returns A tuple containing a boolean that signals whether the value was serializable and
+ * a number of bytes that it will use when serialized to JSON.
+ */
+export function getJsonSerializableInfo(
+  value: unknown,
+  skipSizing = false,
+): [boolean, number] {
```

Comment:
For the sake of documentation, can you name these tuple parameters?

Reply:
Added some names that will explain it, to the `@returns` dosctring part. I hope that helps. Let me know if you have any other idea for it.

Reply:
I meant something like this:
```suggestion
): [isValid: boolean, plainTextSizeInBytes: number] {
```

Reply:
Ok, good. Added in the next commit.